src/payment/strategies/digitalriver/digitalriver-script-loader.ts

```diff
@@ -11,16 +9,12 @@ export default class DigitalRiverScriptLoader {
         private _window: DigitalRiverWindow = window
     ) {}
 
-    async load(digitalRiverPublicApiKey: string, locale: string): Promise<DigitalRiverJS> {
+    async load(): Promise<DigitalRiverWindow> {
```

Comment:
Rather than returning the window, could you instead return `DigitalRiverClass` ?